pkg/storage/fs/cephfs/cephfs.go

```diff
@@ -615,18 +624,208 @@ func (fs *cephfs) UpdateStorageSpace(ctx context.Context, req *provider.UpdateSt
 	return nil, errtypes.NotSupported("unimplemented")
 }
 
+var fnvHash hash.Hash32 = fnv.New32a()
+
+func getHash(s string) uint64 {
+	fnvHash.Write([]byte(s))
+	defer fnvHash.Reset()
+	buf := bytes.NewReader(fnvHash.Sum(nil))
+	var res uint32
+	binary.Read(buf, binary.BigEndian, &res)
+	return uint64(res)
```

Comment:
```suggestion
	return uint64(fnvHash.Sum32())
```

I think [`Sum32()`](https://pkg.go.dev/hash/fnv#New32a) should do the same as these lines.

Reply:
Great, I was a bit hesitant with that much "boiler-plate" code